plugins/radar-android-google-places/src/main/java/org.radarbase.passive.google.places/GooglePlacesService.kt

```diff
@@ -35,10 +36,15 @@ import java.util.concurrent.atomic.AtomicBoolean
 import kotlin.math.pow
 
 class GooglePlacesService: SourceService<GooglePlacesState>() {
-    private val apiKey: ChangeRunner<String> = ChangeRunner(GOOGLE_PLACES_API_KEY_DEFAULT)
+    private val apiKey: ChangeRunner<String> = ChangeRunner()
     lateinit var preferences: SharedPreferences
-    val broadcaster = LocalBroadcastManager.getInstance(this)
+    private var _broadcaster: LocalBroadcastManager? = null
+    val placesClientCreated = AtomicBoolean(false)
+    val broadcaster: LocalBroadcastManager?
+        get() = _broadcaster
```

Comment:
if you just want to keep the setting of variable private you can just use `private set`. Then you don't need to have separate variable to expose as getter.

```kt
var broadcaster: LocalBroadcastManager? = null
        private set
```